clients/common/hipblas_template_specialization.cpp

```diff
@@ -1212,6 +1212,62 @@ hipblasStatus_t hipblasZdrotCast(hipblasHandle_t       handle,
     return hipblasZdrot(handle, n, (hipDoubleComplex*)x, incx, (hipDoubleComplex*)y, incy, c, s);
 }
 
+hipblasStatus_t hipblasCrotCast_64(hipblasHandle_t       handle,
+                                   int64_t               n,
+                                   hipblasComplex*       x,
+                                   int64_t               incx,
+                                   hipblasComplex*       y,
+                                   int64_t               incy,
+                                   const float*          c,
+                                   const hipblasComplex* s)
+{
+    return hipblasCrot_64(
+        handle, n, (hipComplex*)x, incx, (hipComplex*)y, incy, c, (const hipComplex*)s);
+}
+
```

Comment:
Wonder if you should split this into L1, L2, L3, solver files?  Not sure if is one of the slower ones or not....

Reply:
I can play around with this once everything is in.